Gopkg.toml

```diff
@@ -31,46 +29,43 @@
 [[constraint]]
   name = "github.com/dgrijalva/jwt-go"
   version = "3.2.0"
-#  version = "3.1.0"
 
 [[constraint]]
   name = "github.com/docker/docker"
-  version = "~17.03.2-ce"
+  revision = "ed7b6428c133e7c59404251a09b7d6b02fa83cc2" # v18.05.0-ce
 
-[[constraint]]
-  name = "github.com/docker/go-connections"
-  version = "0.3.0"
-#  version = "0.2.1"
+[[override]]
+  name = "github.com/docker/distribution"
+  revision = "edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c" # v18.05.0-ce
+
+[[override]]
+  name = "github.com/vmware/vic"
```

Comment:
I'm guessing this is pulled in on this pr by accident?

Reply:
No, for whatever reason, `dep` decided to complain this time about `vic` and their dependency on `Sirupsen/logrus` where we already have lowercase `sirupsen/logrus` vendored. This overrides their version to pull from a source where the imports have been updated.